web/controllers/auth_controller.ex

```diff
@@ -1,17 +1,18 @@
 defmodule Janitor.AuthController do
+  import Janitor.DBHelpers
   use Janitor.Web, :controller
   use Timex
   alias Janitor.User
-
+  
```

Comment:
You can set up your editor to clean up whitespace in empty lines. It's a good practice to do so.